pallets/gear/src/tests.rs

```diff
@@ -79,6 +79,42 @@ use utils::*;
 
 type Gas = <<Test as Config>::GasProvider as common::GasProvider>::GasTree;
 
+#[test]
+fn calculate_gas_returns_not_block_limit() {
```

Comment:
Does it return a value not equal to the limit always or just in some cases? Would be nice to see in the test name something like `never` or `when_bla_bla_bla`

Reply:
> Does it return a value not equal to the limit always or just in some cases? Would be nice to see in the test name something like `never` or `when_bla_bla_bla`

Exact cases are not identified. Atm its a lot of them when message is consumed, but for me its like general-purpose test so see no reason to specify something special in test name